.vscode/settings.json

```diff
@@ -1,3 +1,6 @@
 {
+	// This setting is unnecessary when using >= v3.88.1
```

Comment:
Are comments allowed in VsCode settings Json files?

Reply:
Yep! :D

it has the language "JSON with Comments" in VS Code:

![image](https://github.com/jakemac53/macros_example/assets/1078012/4ec05ac6-863d-4d13-a19f-abe4d3de3520)

Most of my settings files are comments/commented out settings 🙃